instance/tests/test_ansible.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Ansible - Tests
+"""
+
+# Imports #####################################################################
+
+import yaml
+from unittest.mock import call, patch
+
+from instance import ansible
+from instance.tests.base import TestCase
+
+
+# Tests #######################################################################
+
+class YAMLTestCase(TestCase):
+    """
+    Test cases for YAML helper functions
+    """
+    def setUp(self):
+        self.yaml_dict1 = {
+            'testa': 'firsta with unicode «ταБЬℓσ»',
+            'testb': 'firstb',
+            'test_dict': {
+                'foo': 'firstfoo',
+                'bar': 'firstbar',
+            },
+        }
+        self.yaml_dict2 = {
+            'testb': 'secondb with unicode «ταБЬℓσ2»',
+            'testc': 'secondc',
+            'test_dict': {
```

Comment:
@antoviaque somewhere at the top I mentioned that `yaml_merge` likely won't work with nested dicts. So:

* If decided that such behavior is correct - add test to excercise it.
* If decided to update so that it recursively merges any depth of nested dicts - add test for it.